test/sqllogictest/aggregates.slt

```diff
@@ -587,6 +587,12 @@ SELECT JSONB_OBJECT_AGG(a,b) from (SELECT TRUE::text as a, FALSE::text as b FROM
 ----
 {"true":"false"}
 
+# the remainder of this this test contains window functions, so we should enable them first
```

Comment:
I think the entire sqllogictest should run with this enabled, so that it does not have to be enabled per-test. Maybe there is some constructor where this can be enabled once for the entire sqllogictest binary.